doc/icewm.adoc

```diff
@@ -1037,6 +1037,10 @@ List of battery names ignore.
 +
 Show CPU frequency in CPU status tool tip.
 
+* `NetStatusShowOnlyRunning = 0`
++
+Show network status for running devices only.
```

Comment:
Please explain of what running means (or even change the name). Connected? Data exchanged? What's the perceived difference to not setting this?

Reply:
I think "connected" is a good practical definition. How about:

"Show network status only for connected devices, such as an active ethernet link or associated wireless interface. If false, any network interface that has been brought up will be displayed."

Reply:
Sounds good!